base/docs/helpdb/Base.jl

```diff
@@ -2265,11 +2265,11 @@ sortperm
 """
     mod2pi(x)
 
-Modulus after division by 2pi, returning in the range \[0,2pi).
+Modulus after division by `2pi`, returning in the range `\[0,2pi)`.
```

Comment:
Use backticks both before and after the `2pi`, no `''`s:

```
Modulus after division by ``2π``, returning in the range ``[0, 2π)``.
```

Might as well use a unicode pi as well, and the `\` isn't needed.